setup.py

```diff
@@ -1,20 +1,10 @@
-
-#from distutils.core import setup
+# -*- coding: utf-8 -*-
 from setuptools import setup
-import sys
-
-REQUIRES = ['oauth2']
-if sys.version_info < (2 , 6):
-    REQUIRES.append('simplejson')
 
-setup(name='cartodb',
-      author = 'Javi Santana',
-      author_email = 'jsantana@vizzuality.com',
-      description = 'client to access cartodb api',
-      version='0.8.2',
-      url='https://github.com/Vizzuality/cartodb',
-      install_requires=REQUIRES,
-      packages=['cartodb'],
-      requires = REQUIRES,
-      test_suite='test.client'
-)
+setup(name="carto",
+      author="Daniel Carrión",
+      author_email="daniel@cartodb.com",
+      description="SDK around CartoDB's APIs",
+      version="1.0.0",
```

Comment:
Given that this is a incredible backwards compatible change, instead of marking this as version v1.0.0, and taking into account that there might be some bugs, I would recommend changing the version to a dev/rc candidate, so that people  don't mistakenly upgrade to it with bugs.

The string I would use would be 1.0.0a1 to mark it as alpha, and not have it upgraded to by pip by default.

More on versioning: https://www.python.org/dev/peps/pep-0440/#examples-of-compliant-version-schemes

More on pip install's mechanism: https://pip.pypa.io/en/stable/reference/pip_install/#pre-release-versions